.github/workflows/pr-textlint.yml

```
@@ -214,7 +216,8 @@ jobs:
             }
             console.log("call pulls.list:", pulls_list_params)
             const res = await github.pulls.list(pulls_list_params)
-            await Promise.all(res.data.map(async data => {
+
+            for (const data of res.data) {
```

nits. `response` と `res` があってややこしいので判別しやすくしたい

自分ならこう書く

```suggestion
            const pulls = await github.pulls.list(pulls_list_params).then(({ data }) => data)

            for (const pull of pulls) {
```

---

同じAPIのレスポンスは同じ変数名の変数で受け取るように修正: https://github.com/dev-hato/hato-bot/pull/696/commits/817a6796ce741e6948d31c2dfa7d482ca72fef03